docker-compose.yml

```diff
@@ -6,13 +6,40 @@ services:
     ports:
       - 80:80
       - 443:443
+    restart: always
     volumes:
+      - ./src/LinkDotNet.Blog.Web/appsettings.json:/app/appsettings.json
       - /root/.aspnet/DataProtection-Keys:/root/.aspnet/DataProtection-Keys
       - ./Blog.db:/app/Blog.db # Sqlite datebase
       - /root/aspnetapp.pfx:/app/aspnetapp.pfx # ssl certificate
+      - ./Blog.db:/app/Blog.db #SQlite datebase if you use sqlite
```

Comment:
Seems duplicated now? Merge-conflict?

Reply:
I should only accept one when merging my test branch,but I didn't do it correctly . I've corrected it.